### PR TITLE
Added detection of type worker option and aspect ratio

### LINKED ISSUE #2693: About aspect-ratio CSS property
**The question**
Can Modernizr increase the detection of "aspect-ratio" CSS property？

https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio

### LINKED ISSUE #2694: ECMAScript modules in workers
**Which feature do you want to detect?**
Whether a browser can load web workers as modules. This is instantiated with `new Worker('..', { type: 'module' })`.

**Which browsers do support this feature?**
Most modern browsers with a notable exception of Firefox.
https://caniuse.com/mdn-api_worker_worker_ecmascript_modules

**How could it be implemented?**
https://stackoverflow.com/a/62963963

### PR DESCRIPTION
**Pull request by debadutta98**
Hi @rejas @stucox @patrickkettner,

- Added two Feature
Detect [Aspect ratio](https://caniuse.com/mdn-css_properties_aspect-ratio) CSS property
Detect web worker [type](https://caniuse.com/mdn-api_worker_worker_ecmascript_modules) option

Closes #2694 
Closes #2693  

modernizr version-4.0.0-alpha
Node version:- v14.18.0
OS-window 11

Thank you, waiting for your response

### CODE REVIEW

**Comment by debadutta98** [comment]
Hi @rejas  I just updated my solution and it's passed all test cases

## Caniuse
image

## System testing
 + image

Thank you! waiting for your response

File: feature-detects/css/aspectratio.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*!
+{
+  "name": "aspectration css property",
```
Comment (rejas) [inline feature-detects/css/aspectratio.js:3]: typo :-)

File: feature-detects/css/aspectratio.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "notes": [{
+    "name": "MDN Docs",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio"
+  }]
+}
+!*/
+/* DOC
+Detect working status of all aspectratio css property
+*/
+/**
```
Comment (rejas) [inline feature-detects/css/aspectratio.js:18]: no need to split the comments here

File: feature-detects/workers/workertypeoption.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  "notes": [{
+    "name": "MDN Docs",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/Worker/Worker"
+  }]
+}
+!*/
+/* DOC
+Detect working status of all Workeroptions
+*/
+/**
```
Comment (rejas) [inline feature-detects/workers/workertypeoption.js:18]: again, keep the doc comments in one block

**Review by rejas — CHANGES_REQUESTED** [review]
thx, at first glance looks not bad :) small cosmetic stuff, and please revert the changes to the package-lock.json file

File: feature-detects/css/aspectratio.js (hunk 1 above)
Comment (debadutta98) [reply feature-detects/css/aspectratio.js]: Thank you for addressing the issue is resolved 👍

File: feature-detects/css/aspectratio.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/*!
+{
+  "name": "aspectration css property",
+  "property": "aspectratio",
+  "tags": ["css aspectration", "aspect-ratio"],
```
[the remark this answers is not in the document]
Comment (debadutta98) [reply feature-detects/css/aspectratio.js]: @rejas Thank you for addressing the issue is resolved

File: feature-detects/workers/workertypeoption.js (hunk 3 above)
Comment (debadutta98) [reply feature-detects/workers/workertypeoption.js]: Thank you for addressing the issue is resolved 👍

**Comment by debadutta98** [comment]
> thx, at first glance looks not bad :) small cosmetic stuff, and please revert the changes to the package-lock.json file

@rejas Thank you for addressing the issue package-lock.json is successfully reverted at [4372a33](https://github.com/Modernizr/Modernizr/pull/2702/commits/4372a3372598f1e06ac88692767391819a125244)

**Review by rejas — APPROVED** [review]

**Comment by rejas** [comment]
Thx for the PR!

**Pull request merged**